variants/PX_HER0/variant.h

```diff
@@ -0,0 +1,231 @@
+ *******************************************************************************
+ */
+
+#ifndef _VARIANT_ARDUINO_STM32_
+#define _VARIANT_ARDUINO_STM32_
+
+#ifdef __cplusplus
+extern "C" {
+#endif // __cplusplus
+
+/*----------------------------------------------------------------------------
+ *        Pins
+ *----------------------------------------------------------------------------*/
+
+// Mapping of STM32 GPIO pin to digital pin number
+#define PA6                 0      /* PX_GPIO0 */
+#define PA5                 1      /* PX_GPIO1 */
+#define PA1                 2      /* PX_GPIO2 */
+#define PA0                 3      /* PX_GPIO3 */
+#define PC3                 4      /* PX_GPIO4 */
+#define PC2                 5      /* PX_GPIO5 */
+#define PC1                 6      /* PX_GPIO6 */
+#define PB7                 7      /* PX_GPIO7 */
+#define PH0                 8      /* PX_USR_LED */
+#define PC0                 9      /* PX_LCD_BTN_1_LT */
+#define PH1                 10     /* PX_LCD_BTN_2_RT */
+#define PC13                11     /* PX_LCD_BTN_3_UP */
+#define PC12                12     /* PX_LCD_BTN_4_DN */
+#define PA15                13     /* PX_LCD_BTN_5_YES */
+#define PC9                 14     /* PX_LCD_BTN_6_NO */
+#define PA8                 15     /* PX_USB_DET */
+#define PB11                16     /* PX_LCD_RS */
+#define PB12                17     /* PX_LCD_BACKLIGHT */
+#define PC6                 18     /* PX_PWM_BUZZER */
+#define PC7                 19     /* PX_VBAT_MEAS_EN */
+#define PC8                 20     /* PX_PWR_HOLD */
+#define PA9                 21     /* PX_UART1_TX */
+#define PA10                22     /* PX_UART1_RX */
+#define PA2                 23     /* PX_UART2_TX */
+#define PA3                 24     /* PX_UART2_RX */
+#define PC10                25     /* PX_UART4_TX */
+#define PC11                26     /* PX_UART4_RX */
+#define PB8                 27     /* PX_I2C1_SCL */
+#define PB9                 28     /* PX_I2C1_SDA */
+#define PB6                 29     /* PX_SPI1_CS */
+#define PB3                 30     /* PX_SPI1_SCK */
+#define PB5                 31     /* PX_SPI1_MOSI */
+#define PB4                 32     /* PX_SPI1_MISO */
+#define PB2                 33     /* PX_SPI2_CS_SF */
+#define PB10                34     /* PX_SPI2_CS_LCD */
+#define PD2                 35     /* PX_SPI2_CS_SD */
+#define PB13                36     /* PX_SPI2_SCK */
+#define PB15                37     /* PX_SPI2_MOSI */
+#define PB14                38     /* PX_SPI2_MISO */
+#define PA4                 39     /* PX_DAC1 */
+#define PB0                 40     /* PX_ADC0 */
+#define PC5                 41     /* PX_ADC1 */
+#define PC4                 42     /* PX_ADC2 */
+#define PA7                 43     /* PX_ADC3 */
+#define PB1                 44     /* PX_ADC_VBAT */
+
+// Mapping of net name to digital pin number
+#define PX_GPIO0            0
```

Comment:
I advise to use `PYn` notation for better readability.
But it's up to you :wink:
```suggestion
#define PX_GPIO0            PA6
```